src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,49 @@
+import { useMemo } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person,
+  people: Person[];
+  handleSlugUser: (value: string) => void;
+  isParent: 'no' | 'mother' | 'father';
+  allParams: string;
+};
+
+export const PersonLink = ({
+  person,
+  handleSlugUser,
+  people,
+  isParent,
+  allParams,
+}:Props) => {
+  const findPerson = useMemo(() => {
+    if (isParent === 'no') {
+      return person;
+    }
+
+    let parent = people.find(one => one.name === person.motherName);
+
+    if (isParent === 'father') {
+      parent = people.find(one => one.name === person.fatherName);
+    }
+
+    if (!parent) {
+      return person;
+    }
+
+    return parent;
+  }, []);
+
+  return (
+    <Link
+      className={findPerson.sex === 'f'
+        ? 'has-text-danger'
+        : ''}
```

Comment:
You already have the comment about classnames. Would be great to fix all similar mistakes yourself 🤗